packages/flutter/lib/src/rendering/box.dart

```diff
@@ -996,6 +1009,8 @@ abstract class RenderBox extends RenderObject {
       return true;
     });
   }
+
+  /// In debug mode, paints a border around this render box.
```

Comment:
mention that this is called for every render box if debugPaintSizeEnabled is true